ladybug/datacollection.py

```diff
@@ -145,8 +149,77 @@ def data(self):
         return self._data
 
     def duplicate(self):
-        """Duplicate current data list."""
-        return DataCollection(self.data, self.header)
+        """Return a copy of the current data list."""
+        return DataCollection(self.data, self.header.duplicate())
+
+    def is_in_range(self, lower=float('-inf'), upper=float('+inf'),
```

Comment:
@chriswmackey, how do you imagine this method to be used? If this is something that we want to use for validating data when creating a datacollection I suggest to make this as an internal method.

Reply:
@mostaphaRoudsari ,
I made this exposed because there are a lot of times when someone would want to run a specific range check on the data because of where the data is going or coming from but this should, by no means, be a check that is run automatically under the hood every time you create the DataCollection.  For example, if I am post processing the results of chilled water temperature out of EnergyPlus and I want to make sure that the water is never going below 0 C (since that obviously means that pipes are freezing), I could run this method.   Or, if I know a certain comfort model cannot handle MRT values above 60 C (which can be possible in extremely intense sun), this method is useful here as well.  For this reason, I believe that this method should be exposed.

Reply:
I see how you see this to be used. I think a more common way of doing this is to add a method that returns `bounds` of the values. `is_in_range` sounds like a method that you send a value to see if it is in an acceptable range. For cases that you mentioned you can check the bounds and see if the range makes sense.

Reply:
Good idea.  Done!